editor/connections_dialog.cpp

```diff
@@ -108,17 +108,26 @@ class ConnectDialogBinds : public Object {
  * Signal automatically called by parent dialog.
  */
 void ConnectDialog::ok_pressed() {
-	if (dst_method->get_text() == "") {
+	String method_name = dst_method->get_text();
+
+	if (method_name == "") {
 		error->set_text(TTR("Method in target node must be specified."));
 		error->popup_centered();
 		return;
 	}
+
+	if (method_name.strip_edges().find_char(' ') > 0) {
```

Comment:
There's more than just spaces making a string an invalid identifier. Use `String::is_valid_identifier` :)

Reply:
You're correct! ;)